content/300-guides/200-deployment/201-serverless/400-deploy-to-aws-lambda.mdx

```diff
@@ -1,23 +1,33 @@
 tocDepth: 3
 ---
 
 <TopBlock>
 
-This guide explains how to avoid common issues when deploying a project using Prisma with [Serverless Framework](https://www.serverless.com/framework/).
+This guide explains how to avoid common issues when deploying a project using Prisma to [AWS Lambda](https://aws.amazon.com/lambda/).
 
-The Serverless Framework simplifies deployment to AWS Lambda. It provides a CLI that helps with workflow automation and AWS resource provisioning. While Prisma works well with the Serverless Framework "out of the box", there are a few improvements that can be made within your project to ensure a smooth deployment and performance. There is also additional configuration that is needed if you are using the [`serverless-webpack`](https://www.npmjs.com/package/serverless-webpack) or [`serverless-bundle`](https://www.npmjs.com/package/serverless-bundle) libraries.
+While a framework is not required to deploy to AWS Lambda, this guide covers deploying with [AWS SAM](https://aws.amazon.com/serverless/sam/), [Serverless Framework](https://www.serverless.com/framework/), and [SST](https://sst.dev/).
+
+AWS SAM is a open-source framework from AWS that can be used in the creation of serverless applications. AWS SAM includes the [AWS SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-reference.html#serverless-sam-cli) which you will use to build, test, and deploy your application.
+
+Serverless Framework provides a CLI that helps with workflow automation and AWS resource provisioning. While Prisma works well with the Serverless Framework "out of the box", there are a few improvements that can be made within your project to ensure a smooth deployment and performance. There is also additional configuration that is needed if you are using the [`serverless-webpack`](https://www.npmjs.com/package/serverless-webpack) or [`serverless-bundle`](https://www.npmjs.com/package/serverless-bundle) libraries.
+
+SST provides tools that make it easy for developers to define, test, debug, and deploy their application. Prisma works well with SST, but must be configured so that your schema is correctly packaged by SST.
 
 </TopBlock>
 
-## Prerequisites
+## General AWS Lambda guidance
 
-This guide assumes you have an existing project using both Serverless Framework and Prisma. For a guide on how to get started with Serverless Framework, we recommend [Setting Up Serverless Framework With AWS](https://www.serverless.com/framework/docs/getting-started).
+This section covers changes you will need to make to your application, regardless of framework. After following this guidance, follow the steps for your framework.
 
-## Binary targets in <inlinecode>schema.prisma</inlinecode>
+- [AWS SAM guidance](#aws-sam-guidance)
+- [Serverless Framework guidance](#serverless-framework-guidance)
+- [SST guidance](#sst-guidance)
+
+### Binary targets in <inlinecode>schema.prisma</inlinecode>
```

Comment:
```suggestion
### Define binary targets in Prisma Schema</inlinecode>
```
Formulated more like a change, and also get rid of specific file name (which some users might not use and have configured differently)